profiles/migrations/0005_add_extrafield.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 4.2.9 on 2024-04-15 22:39
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django_extensions.db.fields
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('profiles', '0005_add_extrafields'), ('profiles', '0006_alter_extrafields_value'), ('profiles', '0007_rename_extrafields_extrafield_and_more')]
```

Comment:
@ryancheley Oops, we missed this one and I'll push a fix.

Reply:
It's here https://github.com/djangopackages/djangopackages/compare/profiles-add-extra-fields?expand=1

Reply:
🤦‍♂️ 

thanks @jefftriplett!

Reply:
@ryancheley all good. Code reviews are hard 😬